### ISSUE TITLE
Login after verify email redirects back to verify email 

### ISSUE THREAD

**Issue by avishkar58** [deep-learning-indaba/Baobab#607]
This unintended behaviour was introduced after #581

**Comment by avishkar58** [comment]
@KaleabTessera do you know a quick fix for this?

**Comment by KaleabTessera** [comment]
Yeah I will sort it out later today @avishkar58 .

**Comment by KaleabTessera** [comment]
Going to need a little bit more time with this one. React's go back feature uses your browser history so if you come from another site, it redirects there. Will add check to see if history from another site.

**Comment by avishkar58** [comment]
Cool, let's disable it until you've figured it out then?

**Comment by ChrisCurrin** [comment]
> Going to need a little bit more time with this one. React's go back feature uses your browser history so if you come from another site, it redirects there. Will add check to see if history from another site.

or check to see if verifyEmail is in the params of the URL (if you copy the link and paste it in a new tab, instead of clicking on the link); would that work?

**Comment by KaleabTessera** [comment]
@ChrisCurrin  Yeah that should work for this situation, not sure what other scenarios this would happen in though. 

Do you want to give this a go? 

The line doing this is `this.props.history.goBack()`

You can use `this.props.location` + javascript's `incudes` function to check if the current url has verify email.

**Comment by avishkar58** [comment]
@KaleabTessera are you able to add a quick fix for this please? Will cause issues in our testing otherwise...

**Comment by KaleabTessera** [comment]
Sorry @avishkar58. Will get this in today.
